plugins/bf1942/bf1942.cpp

```diff
@@ -59,7 +58,7 @@ static int fetch(float *avatar_pos, float *avatar_front, float *avatar_top, floa
 
 	//peekProc(contextptr, ccontext, 128);
 
-	ok = peekProc((BYTE *) 0x00976274, avatar_pos, 12) &&
+	ok = peekProc((procptr32_t) 0x00976274, avatar_pos, 12) &&
```

Comment:
Cast should not be necessary?

(Goes for a lot of places in throughout the PR)